bigquery/unit_tests/test__helpers.py

```diff
@@ -491,6 +491,363 @@ def test_instance_setter_w_bad_udfs(self):
         self.assertEqual(instance.udf_resources, [])
 
 
+class Test_AbstractQueryParameter(unittest.TestCase):
+
+    @staticmethod
+    def _get_target_class():
+        from google.cloud.bigquery._helpers import AbstractQueryParameter
```

Comment:
1. That's fine, we don't need to hold this PR up
1. It would be very easy for you to just add the one extra character
1. There will never be "related" PRs